env.example

```diff
@@ -9,6 +9,12 @@ export DNS_RESOURCEGROUP=dns
 # Set the OpenShift version (3.10) (mandatory)
 export DEPLOY_VERSION=3.10
 
+# Set DEV_PULL_SECRET when running pre-release/dev build of RHEL
+# You can get token string from docker login and cat ~/.docker/config.json
+#export DEV_PULL_SECRET=xxxxxxxxx
+# Set OREG_URL to overwrite registry and namespace. Used for testing.
+export OREG_URL='registry.reg-aws.openshift.com/openshift3/ose-${component}:${version}'
```

Comment:
I would avoid exporting this by default and would use it only on test jobs that need to export it (rhel image builds, pr tests).

Reply:
ack.

Reply:
ack.